### PR TITLE
DOP-2322 & DOP-2323 - Implement QuizChoice component layout [Correct]

### PR DESCRIPTION
**Pull request by carooobregon**
**New PR bc had some problems with the staging link on the past one!**
[[old pr]
](https://github.com/mongodb/snooty/pull/449)
Stories/Links:
DOP-2322
DOP-2323

Staging Links:
[Random article I added a quiz widget to
](https://docs-mongodbcom-staging.corp.mongodb.com/master/cloud-docs/carolinao/DOP-2323/mongodb-logs/)
Notes:
Implemented QuizChoice with their respective styling depending on different states: Hovered, Selected, Submitted wrong question, Submitted right question
Sent setState callback functions as props to pass state between components

### CODE REVIEW

**Review by rayangler — APPROVED** [review]
LGTM!

@casthewiz you had some comments on the previous PR. Just wanted to check in with you that all changes since then look good

File: src/components/QuizChoice.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    <DescriptionBody>
+      {description.map((node, i) => (
+        <ComponentFactory nodeData={node} key={i} />
+      ))}
+    </DescriptionBody>
+  );
+};
+
+const CorrectChoice = () => <SyledIcon glyph="Checkmark" fill={uiColors.green.base} size="small" />;
+const IncorrectChoice = () => <SyledIcon glyph="X" fill={uiColors.gray.base} size="small" />;
```
Comment (casthewiz) [inline src/components/QuizChoice.js:58]:
```suggestion
const CorrectChoice = () => <StyledIcon glyph="Checkmark" fill={uiColors.green.base} size="small" />;
const IncorrectChoice = () => <StyledIcon glyph="X" fill={uiColors.gray.base} size="small" />;
```

File: src/components/QuizChoice.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    props.selectedThisChoice && !props.hasSubmitted ? `${uiColors.gray.dark3}` : `white`};
+  border-color: black;
+  border-radius: 50%;
+  display: inline-block;
+  border-style: solid;
+  border-width: thin;
+  margin-right: 16px;
+`;
+
+const SyledIcon = styled(Icon)`
```
Comment (casthewiz) [inline src/components/QuizChoice.js:28]:
```suggestion
const StyledIcon = styled(Icon)`
```

**Review by casthewiz — COMMENTED** [review]
Just a small typo fix 😄

**Review by casthewiz — APPROVED** [review]

**Review by allisonmui9 — APPROVED** [review]
so cool caro! amazing job. thank you!! LGTM! :)

**Pull request merged**